spec/interactions/claim_contribution_spec.rb

```diff
@@ -1,32 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe ClaimContribution do
-  before do
-    allow(MatchContribution).to receive(:run!)
-    allow(EmailPeer).to receive(:run!)
-  end
-
   let(:contribution) { create(:listing) }
   let(:user) { create(:user, :with_person, email: 'user_email@example.com') }
 
-  before do
+  let(:claim_contribution) do
     ClaimContribution.run!(
       contribution: contribution.id,
       current_user: user,
+      organization: build(:organization),
       peer_alias: 'alias',
       message: 'message',
     )
   end
 
-  it "matches contribution and sends an email to peer", :aggregate_failures do
-    expect(MatchContribution).to have_received(:run!)
```

Comment:
Do we use this elsewhere?

Reply:
Do you mean `MatchContribution`?

Reply:
I meant the first, but was wondering re both while reviewing from my phone. :)

Reply:
Thanks! If you're asking about testing, this example is replaced by the two below and changed from being mocked to checking the side-effects of `MatchContribution` and `EmailPeer`.

As for production code, both are only used in `ClaimContribution`.